.vsconfig

```diff
@@ -0,0 +1,13 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
+    "Microsoft.Net.Component.4.7.2.SDK",
+    "Microsoft.Net.Component.4.7.2.TargetingPack",
+    "Microsoft.VisualStudio.Component.Azure.Storage.Emulator",
```

Comment:
Anyone know where this is used?

Reply:
Far as I can tell, it isn't. Might have been added in prep for tests using Azure Storage that never materialized or were removed when we dropped Microsoft.AspNetCore.DataProtection.AzureStorage.

Reply:
Removed it